.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: cpp
+sudo: false
+
+env:
+  global:
+    - NODE_VERSION="6.1.0"
+
+os:
+  - linux
+  - osx
+
+before_install:
+  - rm -rf ~/.nvm
+  - git clone https://github.com/creationix/nvm.git ~/.nvm
+  - source ~/.nvm/nvm.sh
+  - nvm --version
+  - nvm install $NODE_VERSION
+  - node --version
+  - npm --version
+  - pip install --user cpplint
```

Comment:
Also, what do you think about using [gdams/node-cpplint](https://github.com/gdams/node-cpplint) as a dev-dependency instead of an external dependency via pip?